state_dim: 4
control_dim: 2
measurement_dim: 4
process_noise: [0.1, 0.1, 0.05, 0.05]
R1: [0.1, 0.1, 0.05, 0.05]
R2: [0.1, 0.1, 0.05, 0.05]
init_covar: [10.0, 10.0, 10.0, 10.0]
X_init: [0.,0.,0.,0.]
odom1_topic: "/odom1"
odom2_topic: "/odom2"
out_odom_topic: "/odom_fitered"
